Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -396,14 +396,6 @@ def audit_deps
         end
 
         case dep.name
-        when *BUILD_TIME_DEPS
-          next if dep.build? || dep.run?
-          problem <<-EOS.undent
-            #{dep} dependency should be
-              depends_on "#{dep}" => :build
-            Or if it is indeed a runtime dependency
-              depends_on "#{dep}" => :run
```

Comment:
This `problem` seems to have been dropped by this commit. It was also adjusted in #1290. Was it dropped on purpose? I'll open a pull request to continue the discussion